demos/multi_object_classification_pipeline/python/label_mapping.py

```diff
@@ -0,0 +1,1000 @@
+label_mapping = {0: 'tench, Tinca tinca',
```

Comment:
Is it ImageNet? Perhaps we have such file in `common` directory?

Reply:
Done